assets/javascripts/tests.js

```diff
@@ -81,27 +81,14 @@ function renderTestName(data, type, row) {
 
     var deps = row.deps;
     if (deps) {
-        var parents = deps.parents;
-        var children = deps.children;
-        var depsTooltip = [];
-        depsTooltip.quantify = function(quandity, singular, plural) {
-            if (quandity) {
-                this.push([quandity, quandity === 1 ? singular : plural].join(' '));
-            }
-        };
-        depsTooltip.quantify(parents.Chained.length, 'chained parent', 'chained parents');
-        depsTooltip.quantify(parents['Directly chained'].length, 'directly chained parent', 'directly chained parents');
-        depsTooltip.quantify(parents.Parallel.length, 'parallel parent', 'parallel parents');
-        depsTooltip.quantify(children.Chained.length, 'chained child', 'chained children');
-        depsTooltip.quantify(children['Directly chained'].length, 'directly chained child', 'directly chained children');
-        depsTooltip.quantify(children.Parallel.length, 'parallel child', 'parallel children');
-        if (depsTooltip.length) {
-            var childrenToHighlight = children.Parallel.concat(children.Chained, children['Directly chained']);
-            var parentsToHighlight = parents.Parallel.concat(parents.Chained, parents['Directly chained']);
-            html += ' <a href="/tests/' + row.id + '" title="' + depsTooltip.join(', ') + '"' +
-                highlightJobsHtml(childrenToHighlight, parentsToHighlight) +
+        var dependencyResult = showJobDependency(deps);
+        var dependencyHtml = '';
+        if (dependencyResult['title'] !== undefined) {
```

Comment:
I think this should also be `dependencyResult.title`, isn't it?

Reply:
I guess it isn't that important.